README.md

```diff
@@ -56,7 +56,11 @@ Follow instructions in https://kafka.apache.org/quickstart
 ### Running a particular unit/integration test with log4j output ###
 Change the log4j setting in either `clients/src/test/resources/log4j.properties` or `core/src/test/resources/log4j.properties`
```

Comment:
I think there are many other `log4j.properties` needed to be modified if you are running tests for that project, not just these 2 files. For example, when you are running tests in `TopicCommandTest`, you should update: `tools/src/test/resources/log4j.properties`, instead of the files listed here. I'm thinking we should rephrase it to a more general way. Maybe:

By default, there will be only small number of logs output while testing. You can adjust it by changing the `log4j.properties` in the test project directory. 
For example, if you want to see more logs for clients project tests, you can modify the line in `clients/src/test/resources/log4j.properties` to `log4j.logger.org.apache.kafka=INFO` and then run: ....


WDYT?

Reply:
Thanks for reviewing my PR!
Agree with you that the original phrase is a bit unclear and misleading. I think the version you suggested is much clearer and easy to understand, let me revise base on that!